src/main/java/marquez/db/mappers/ExtendedRunRowMapper.java

```diff
@@ -34,20 +37,26 @@
   @Override
   public ExtendedRunRow map(@NonNull ResultSet results, @NonNull StatementContext context)
       throws SQLException {
+    Set<String> columnNames = MapperUtils.getColumnNames(results.getMetaData());
+
     return new ExtendedRunRow(
         uuidOrThrow(results, Columns.ROW_UUID),
         timestampOrThrow(results, Columns.CREATED_AT),
         timestampOrThrow(results, Columns.UPDATED_AT),
         uuidOrThrow(results, Columns.JOB_VERSION_UUID),
         uuidOrThrow(results, Columns.RUN_ARGS_UUID),
-        uuidArrayOrThrow(results, Columns.INPUT_VERSION_UUIDS),
+        columnNames.contains(Columns.INPUT_VERSION_UUIDS)
+            ? uuidArrayOrThrow(results, Columns.INPUT_VERSION_UUIDS)
+            : ImmutableList.of(),
         timestampOrNull(results, Columns.NOMINAL_START_TIME),
         timestampOrNull(results, Columns.NOMINAL_END_TIME),
         stringOrNull(results, Columns.CURRENT_RUN_STATE),
-        timestampOrNull(results, Columns.STARTED_AT),
+        columnNames.contains(Columns.STARTED_AT)
+            ? timestampOrNull(results, Columns.STARTED_AT)
+            : null,
         uuidOrNull(results, Columns.START_RUN_STATE_UUID),
-        timestampOrNull(results, Columns.ENDED_AT),
+        columnNames.contains(Columns.ENDED_AT) ? timestampOrNull(results, Columns.ENDED_AT) : null,
```

Comment:
_Minor_: Wouldn't` timestampOrNull()` return `null` if not present?  Wondering if we need the extra check.

Reply:
timestampOrNull will throw an exception if the column doesn't exist

Reply:
A larger refactor would be to pass in the column names to every function, which i did in https://github.com/henneberger/marquez/blob/feature/openlineage/src/main/java/marquez/db/mappers/RunMapper.java 

We can bring that paradigm over

Reply:
Nice. That's definitely cleaner 👍

Reply:
Lets backlog it and we can migrate it in a different commit